cluster/juju/layers/kubernetes-master/templates/kube-system-rbac.yaml

```diff
@@ -0,0 +1,14 @@
+# This role binding allows "kube-system:default"  admin privleged access
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: add-on-cluster-admin
+  namespace: kube-system
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Do you think we should have "cluster-admin" as a constant in the .py file and render this file accordingly?